win32/signal.c

```diff
@@ -21,9 +21,10 @@
 
 #include "win32/console.h"
 
-ZEND_TLS zval ctrl_handler;
-ZEND_TLS DWORD ctrl_evt = (DWORD)-1;
-ZEND_TLS zend_bool *vm_interrupt_flag = NULL;
+/* true globals; only used from main thread and from kernel callback */
+zval ctrl_handler;
+DWORD ctrl_evt = (DWORD)-1;
+zend_bool *vm_interrupt_flag = NULL;
```

Comment:
Please make these static. Other than that, looks good.

Thanks.

Reply:
Oh, of course. Thanks!